include/circt/Dialect/SV/SVPasses.td

```diff
@@ -104,7 +104,10 @@ def HWMemSimImpl : Pass<"hw-memory-sim", "ModuleOp"> {
                 "false",
     "ignore the read enable signal, instead of assigning X on read disable">,
     Option<"stripMuxPragmas", "strip-mux-pragmas", "bool", "false",
-            "Do not annotate mux pragmas to memory reads">
+            "Do not annotate mux pragmas to memory reads">,
+    Option<"preventVivadoBRAMMapping", "prevent-vivado-bram-mapping", "bool", "false",
```

Comment:
This isn't a good flag or description.   Please elaborate the description to include when it applies and what it works around.